resultsdbupdater/utils.py

```diff
@@ -176,10 +176,10 @@ def _test_result_outcome(topic, outcome):
 
     Some systems generate outcomes that don't match spec.
 
-    Test outcome is FAILED for messages with "*.error" topic.
+    Test outcome is ERROR for messages with "*.error" topic.
```

Comment:
Should we just drop this line since it's no longer being treated differently than the other states?

Reply:
I think it's worth keeping.